packages/server/src/api/controllers/record.js

```diff
@@ -73,6 +75,8 @@ exports.save = async function(ctx) {
 
   const model = await db.get(record.modelId)
 
+  coerceRecordValues(record, model)
```

Comment:
NAB: it would be a bit cleaner and easier to reason about/test this code if it was pure and immutable, returning the new coerced record rather than just mutating the one passed in

Reply:
done